src/main/java/com/desiato/puresynth/initialization/ScriptTrigger.java

```diff
@@ -0,0 +1,35 @@
+package com.desiato.puresynth.initialization;
+
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+@Component
+public class ScriptTrigger {
```

Comment:
It should not be part of the application. Imagine a situation where you're running it on a real machine on production, and you start inserting admin users into the DB. It's a huge security risk.

The idea of such a script is, that you run it once manually when setting up the development environment. It should never be run in the real environment.

Reply:
ok, got it! This was exactly my doubt. Thank you